### PR TITLE
adding_displayio_example

### PR DESCRIPTION
**Pull request by jposada202020**
tested in a feather ESP32-S2 with an Adafruit BMP280 sensor

### CODE REVIEW

File: examples/bmp280_displayio_simpletest.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+bmp280.sea_level_pressure = 1013.25
+
+# Create two Labels to show the readings. If you have a very small
+# display you may need to change to scale=1.
+tempandpress_output_label = Label(FONT, text="", scale=2)
+altitude_output_label = Label(FONT, text="", scale=2)
+
+# place the labels in the middle of the screen with anchored positioning
+tempandpress_output_label.anchor_point = (0, 0)
+tempandpress_output_label.anchored_position = (4, board.DISPLAY.height // 2)
```
Comment (FoamyGuy) [inline examples/bmp280_displayio_simpletest.py]: The pressure value is running off the screen for me on a feather S3 TFT. 

I was able to get everything fitting nicely by changing the anchored position to:
```
tempandpress_output_label.anchored_position = (4, board.DISPLAY.height // 2 - 40)
``` 
And changing the text in the label to a newline inbetween temperature and pressure
```
f"Temperature:{bmp280.temperature:0.1f} C\nPressure:{bmp280.pressure:0.1f} hPa"
```

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
The pressure value is off the screen for me, there are a few tweaks that can be made to have all the values showing on the display.

File: examples/bmp280_displayio_simpletest.py (hunk 1 above)
Comment (jposada202020) [reply examples/bmp280_displayio_simpletest.py]: That Works

**Comment by jposada202020** [comment]
I think for next examples ill use Scale=2 and try to fit it in the display of the feather, it looks better!
Thanks of rthe suggestion

**Review by FoamyGuy — APPROVED** [review]
Thank you, looks good.

**Pull request merged**